HW8/Task2/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW8.Task2
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        { 
+            try
+            {
+                Client client = new Client() { Experience = 1, Name = "Masha" };
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            finally
+            {
+                Console.WriteLine();
+            }
+            try
+            {
+                Client newClient = new Client() { Experience = 3, Name = "Pasha", Nation = "Russian" };
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            finally
+            {
+                Console.WriteLine();
+            }
+            try
```

Comment:
create private method for all your try catches and provide there your initialization